patches/server/0979-Add-whitelist-events.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SageSphinx63920 <sage@sagesphinx63920.dev>
+Date: Sun, 14 May 2023 12:57:15 +0200
+Subject: [PATCH] Add whitelist events
+
+
+diff --git a/src/main/java/net/minecraft/server/players/UserWhiteList.java b/src/main/java/net/minecraft/server/players/UserWhiteList.java
+index a6c70decad1a496595944a5dd2aeeb387757fef3..f97fda57c84d52ea1c6e9485f289cfb0b16e46da 100644
+--- a/src/main/java/net/minecraft/server/players/UserWhiteList.java
++++ b/src/main/java/net/minecraft/server/players/UserWhiteList.java
+@@ -2,6 +2,8 @@ package net.minecraft.server.players;
+ 
+ import com.google.gson.JsonObject;
+ import com.mojang.authlib.GameProfile;
++import io.papermc.paper.event.player.PlayerWhitelistUpdateEvent;
++
```

Comment:
Don't think this import should be here?

Reply:
Yeah IntelliJ auto imported. I used FQN in the calls